data/schema.yaml

```diff
@@ -106,42 +105,49 @@ components:
           type: string
           format: date-time
           example: 2017-12-21T17:32:28Z
-        MetaDescription:
+        metaDescription:
           type: boolean
           example: null
         model:
           type: object
           properties:
             blocks:
```

Comment:
This is the same as `model/blocks` nested inside this definition of model. Therefore we should be able to change this to be:
```
    model:
        - $ref: '#/components/schemas/blocks'
```

Reply:
Cheers for sorting, github hasn't noticed the diff changed

Reply:
👍